modules/Employees/Employee.php

```diff
@@ -295,6 +295,29 @@ function hasCustomFields()
         //return result of search for custom fields
         return $userCustomfields;
     }
+
+    /**
+	 * Override the original save function,
+	 * for checking first is it same user as employee
+	 * and disable to save any employee data for others.
+	 * (admin user is an exception)
+	 *
+     * @param bool $check_notify
+     * @return bool|string
+     */
+    public function save($check_notify = false)
+    {
+        global $current_user;
+        if ($current_user->id) {
+            if (!is_admin($current_user)) {
+                if ($this->id && $current_user->id != $this->id) {
+                    $GLOBALS['log']->fatal("You can change only your own employee data.");
```

Comment:
Should this be a warn or a security instead of a fatal error?

As nothing is has broken down, instead it is a security violation.

Also It might be worth changing the message to include the name of the user, so that a administrator can see who is causing this issue.